cockatrice/src/replay_timeline_widget.cpp

```diff
@@ -62,7 +62,11 @@ void ReplayTimelineWidget::paintEvent(QPaintEvent * /* event */)
 
 void ReplayTimelineWidget::mousePressEvent(QMouseEvent *event)
 {
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+    int newTime = static_cast<int>((qint64)maxTime * (qint64)event->position().x() / width());
```

Comment:
all this casting is very pointless, this could just be a qint64

Reply:
I'm not fully certain why it does this, but I'm not sure of a better chance as it might be a larger change required across the system